diff --git a/node_modules/@supabase/realtime-js/package.json b/node_modules/@supabase/realtime-js/package.json
index 0855e6a..0c23930 100644
--- a/node_modules/@supabase/realtime-js/package.json
+++ b/node_modules/@supabase/realtime-js/package.json
@@ -39,9 +39,7 @@
   },
   "dependencies": {
     "@supabase/node-fetch": "^2.6.14",
-    "@types/phoenix": "^1.5.4",
-    "@types/ws": "^8.5.10",
-    "ws": "^8.18.0"
+    "@types/phoenix": "^1.5.4"
   },
   "devDependencies": {
     "@arethetypeswrong/cli": "^0.16.2",
diff --git a/node_modules/@supabase/realtime-js/src/RealtimeClient.ts b/node_modules/@supabase/realtime-js/src/RealtimeClient.ts
index 6bfbd21..277a69b 100755
--- a/node_modules/@supabase/realtime-js/src/RealtimeClient.ts
+++ b/node_modules/@supabase/realtime-js/src/RealtimeClient.ts
@@ -1,5 +1,3 @@
-import type { WebSocket as WSWebSocket } from 'ws'
-
 import {
   CHANNEL_EVENTS,
   CONNECTION_STATE,
@@ -63,7 +61,7 @@ export interface WebSocketLikeConstructor {
   ): WebSocketLike
 }
 
-export type WebSocketLike = WebSocket | WSWebSocket | WSWebSocketDummy
+export type WebSocketLike = WebSocket | WSWebSocketDummy
 
 export interface WebSocketLikeError {
   error: any
@@ -210,13 +208,6 @@ export default class RealtimeClient {
         this.conn = null
       },
     })
-
-    import('ws').then(({ default: WS }) => {
-      this.conn = new WS(this.endpointURL(), undefined, {
-        headers: this.headers,
-      })
-      this.setupConnection()
-    })
   }
 
   /**
